int32[] neck
int32[] left_shoulder
int32[] left_elbow
int32[] left_wrist
int32[] right_shoulder
int32[] right_elbow
int32[] right_wrist
